docs/en/stack/ml/anomaly-detection/ml-ad-job-types.asciidoc

```diff
@@ -0,0 +1,117 @@
+[[ml-anomaly-detection-job-types]]
+= {anomaly-detect-cap} job types
+++++
+<titleabbrev>Job types</titleabbrev>
+++++
+
+:keywords: {ml-init}, {stack}, {anomaly-detect}
+
+:description: The description of the different anomaly detection job types. 
+
+{anomaly-jobs-cap} have many possible configuration options which enable you to 
+fine-tune the jobs and cover your use case as much as possible. This page 
+provides a quick overview of the different types of {anomaly-jobs} and their 
+capabilities. The currently available job types are:
+
+* single metric jobs,
+* multi-metric jobs,
+* population jobs,
+* categorization jobs,
+* rare jobs.
```

Comment:
As these are the types of wizard in the ML UI, I think "advanced jobs" should also be in the list.